CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### [Unreleased](https://github.com/redpanda-data/helm-charts/releases/tag/redpanda-FILLMEIN) - YYYY-MM-DD
 #### Added
+* Add RBAC rules for the operator chart so it can manage users
```

Comment:
I approved, but this should be moved to operator chart section. In the https://github.com/redpanda-data/helm-charts/pull/1530 I'm fixing that